tensorboard/plugins/hparams/list_session_groups.py

```diff
@@ -50,32 +50,105 @@ def __init__(
         self._request = request
         self._extractors = _create_extractors(request.col_params)
         self._filters = _create_filters(request.col_params, self._extractors)
+
+    def run(self):
```

Comment:
Just adding a note: The name of this function seems to suggest it would return a run name or run object, but instead it returns a response proto, which is weird to me. Since this isn't added in this PR, we don't need to consider changing it right now. Maybe worth refactoring later.